### PR TITLE
Bookmarklet

### PR DESCRIPTION
**Pull request by kyeotic**
Add option for bookmarklet, which unlike chrome extensions will work in any browser

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ One second to read GitHub code with VS Code.
 
 [Chrome Extension](https://chrome.google.com/webstore/detail/github1s/lodjfmkfbfkpdhnhkcdcoonghhghbkhe/)
 
+Or Save as a bookmarklet (GitHub markdown doesn't allow js links, just copy it into a bookmark) 
+
+```
+javascript: window.location.href = window.location.href.replace('github.com', 'github1s.com')
```
Comment (xcv58) [inline README.md:11]: Could you please use this syntax to make it draggable? https://joeldare.com/creating_a_drag_and_drop_bookmarklet.html#providing-a-bookmarklet-in-markdown

**Comment by bradydowling** [comment]
Not trying to step on toes here but I do know that PRs can sometimes get created and then forgotten so in case that happens, here's one that addresses this https://github.com/conwnet/github1s/pull/54 If this one gets updated, feel free to just close the other

File: README.md (hunk 1 above)
Comment (kyeotic) [reply README.md:11]: I tried and GitHub wont render it as a link. Its a security feature for javascript urls

**Comment by kyeotic** [comment]
@bradydowling I didn't see that, so it's me who should be apologizing. However, I tried making the url draggable and GitHub wont render it as a link. Its a security feature for javascript urls. 

If you check [your own fork](https://github.com/bradydowling/github1s/tree/patch-1) you will see that it doesnt work

File: README.md (hunk 1 above)
Comment (xcv58) [reply README.md:11]: Thanks for sharing this.

**Pull request merged**